sys/ztimer/Makefile.dep

```diff
@@ -27,7 +27,7 @@ endif
 ifneq (,$(filter ztimer,$(USEMODULE)))
   ifneq  (,$(filter xtimer,$(USEMODULE)))
     ifeq (,$(filter ztimer_xtimer_compat,$(USEMODULE)))
-        USEMODULE += xtimer_on_ztimer
+      USEMODULE += xtimer_on_ztimer
```

Comment:
Can you split this out into a separate commit, as this fix of indent is unrelated? Keeping unrelated stuff into separate commits makes life much easier when after merging some bug pops up and specific commits need to be reverted until a long term solution is found.

(Actually, unrelated stuff should be moved into its own PR. But I personally have strong sympathy for not wanting to create a PR for trivial single line fixes.)

Reply:
Done